pyproject.toml

```diff
@@ -148,6 +148,7 @@ randomname = "^0.1.5"
 pluggy = "^1.0.0"
 slack-sdk = "^3.19.2"
 confluent-kafka = "^1.9.2"
+portalocker = "^2.7.0"
```

Comment:
Please double-check that the license of this new dependency is compatible with our use-case(s) in rasa.

Reply:
Its the same license as [numpy](https://github.com/numpy/numpy/blob/main/LICENSE.txt) (BSD-3) so I think its fine 😃

Reply:
Perfect!